backend/agent-manager.ts

```diff
@@ -76,12 +76,15 @@ export class AgentManager {
      * @param url
      * @param username
      * @param password
+     * @param name
+     * @param updatedName
```

Comment:
```suggestion
     * @param updatedName
```
Parameter updatedName is not used in this function, can be removed

Reply:
Thanks for the feedback. Updated in latest commit